contracts/develop/release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release
+---
+
+For versioning we use [setuptools-scm](https://pypi.org/project/setuptools-scm/). This means the version number is
+derived from git tags. To release a new version of the contracts on PyPI or Docker Hub, simply tag a commit with a valid version
+number either via git, or from [github](https://github.com/trustlines-protocol/contracts/releases/new).
+Make sure to update the changelog accordingly and add all changes since the last released version.
+
+## Pre-commit hooks
```

Comment:
Why is that part in the release category? I think it'd be better in the `Start Developing` section

Reply:
yes, you are right.